pkg/kubectl/cmd/convert/convert_test.go

```diff
@@ -90,16 +90,6 @@ func TestConvertObject(t *testing.T) {
 				},
 			},
 		},
-		{
-			name:          "v1beta1 Ingress to extensions Ingress",
-			file:          "../../../../test/fixtures/pkg/kubectl/cmd/convert/v1beta1ingress.yaml",
```

Comment:
this test case should remain... we want `kubectl convert` to continue working if people want to use it to transition to v1

Reply:
restoed